openapi/openapi.yaml

```diff
@@ -1080,6 +1080,35 @@ paths:
               schema:
                 $ref: "#/components/schemas/PythonPackageMetadataResponseError"
 
+  /python/get-imported-packages:
+    get:
+      tags: [PythonPackages]
+      x-openapi-router-controller: thoth.user_api.api_v1
+      operationId: get_imported_packages
+      summary: List imported packages' correct names.
+      parameters:
+        - name: name
```

Comment:
import?

Reply:
> import?

fixed in 0ada6fd042629810b21dcec94b4be45bd7b93e6b